packages/flutter/test/rendering/slivers_block_test.dart

```diff
@@ -67,6 +67,47 @@ class TestRenderSliverBoxChildManager extends RenderSliverBoxChildManager {
   void setDidUnderflow(bool value) { }
 }
 
+class ViewportOffsetSpy extends ViewportOffset {
+  ViewportOffsetSpy(this._pixels);
+
+  double _pixels;
+
+  @override
+  double get pixels => _pixels;
+
+  bool corrected = false;
+
+  @override
+  bool applyViewportDimension(double viewportDimension) => true;
+
+  @override
+  bool applyContentDimensions(double minScrollExtent, double maxScrollExtent) => true;
+
+  @override
+  void correctBy(double correction) {
+    _pixels += correction;
+    corrected = true;
+  }
+
+  @override
+  void jumpTo(double pixels) {
+    // Do nothing, viewport is fixed.
+  }
+
+  @override
+  Future<void> animateTo(
+      double to, {
+        @required Duration duration,
+        @required Curve curve,
+      }) async { }
```

Comment:
It's probably worth repeating the comment from above as to why this is empty.

Reply:
fixed